manifests/g/GLab/GLab/1.52.0/GLab.GLab.installer.yaml

```
@@ -0,0 +1,28 @@
+# Created with komac v2.11.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: GLab.GLab
+PackageVersion: 1.52.0
+InstallerLocale: en-US
+InstallerType: inno
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+ProductCode: '{679C50AE-F48A-46D1-9493-77F095E9884D}_is1'
+ReleaseDate: 2025-01-20
+AppsAndFeaturesEntries:
+- Publisher: GitLab
+  ProductCode: '{679C50AE-F48A-46D1-9493-77F095E9884D}_is1'
+ElevationRequirement: elevatesSelf
+InstallationMetadata:
+  DefaultInstallLocation: '%ProgramFiles%\glab'
+Installers:
+- Architecture: x64
+  InstallerUrl: https://gitlab.com/gitlab-org/cli/-/releases/v1.52.0/downloads/glab_1.52.0_Windows_x86_64_installer.exe
+  InstallerSha256: AA6EE697D57271286EBCCB4FE121DA9FF15DA5490A67CC713591608DAAE519CB

```

```suggestion
  InstallerSha256: BA510493B549B22E788A7AF3BDD9FCFD4009D90A534874A014D47F7B935D88AA
```

(Automated response - build 998.)